deploy/crds/toolchain_v1alpha1_usersignup.yaml

```diff
@@ -11,6 +11,17 @@ spec:
     kind: UserSignup
     plural: usersignups
   scope: Namespaced
+  additionalPrinterColumns:
+    - name: User ID
+      type: string
+      JSONPath: .spec.userID
+      priority: 1
+    - name: Complete
+      type: string
+      JSONPath: ".status.conditions[?(@.type==\"Complete\")].status"
+    - name: Reason
+      type: string
+      JSONPath: ".status.conditions[?(@.type==\"Complete\")].reason"
```

Comment:
Let's maybe also add Approved condition (status + reason) too with `priority: 1` so we don't see it by default.

Reply:
Done.